lib/DDH/UserPage/Generate.pm

```diff
@@ -105,6 +110,19 @@ sub generate {
             }
         );
         close $fh;
+
+        # JSON dump of user's data
+        my $json_build_dir = $self->json_dir( $contributor );
+        open my $jfh, '>:encoding(UTF-8)', "$json_build_dir/index.html" or die();
+        print $jfh $self->xslate->render(
```

Comment:
Wondering if we could just dump index.json alongside index.html without a template? i.e. just the raw JSON as we get it?